R2Bot/src/main.cpp

```diff
@@ -35,6 +36,21 @@ JobHandler getHandlerByJob(std::string job) {
 }
 
 int main(int argc, char *argv[]) {
+	/** Handle arguments */
+	for (int i = 0; i < argc; i++) {
+		if (argv[i] == "--jobs-file") {
```

Comment:
For comparing char *, you'll need `strncmp(argv[i], "--jobs-file", 11)`, or `std::string(argv[i]) == "--jobs-file"`.